susemanager/susemanager.changes

```diff
@@ -1,3 +1,4 @@
+- add gnupg and its dependencies to debian 10 bootstrap repo
```

Comment:
Are the python3 dependencies for gnupug? Or are they dependencies for  `python3-cryptodome`?

Reply:
The only new python package is python3-cryptodome. The list looked like sorted alphabetically so I added gnupg and its dependecies and run a "sort" on the list. This moved also other packages around.

Reply:
True! Sorry for the noise :-)